engines/onboarding/app/policies/onboarding/group_policy.rb

```diff
@@ -7,7 +7,11 @@ def resolve
     end
 
     def bulk?
-      Membership.where(group: record, user: user, role: Membership::ROLES[:admin]).any?
+      Membership.where(
+        group: record,
+        user: user,
+        role: Role.new(:admin).index
```

Comment:
here we should use `::Group::Role` unless we create the `Onboarding::Role` model too (which I wouldn't)